www/src/context/AppContext.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+
+export const AppContext = React.createContext({
+  isSignedIn: undefined,
+  user: undefined,
+});
```

Comment:
Usage note. These are the default values you can provide the context. You do need to specify these default because the context render update happens after the initial mount (at least with me) which cause the component to render empty and throw an error in the browser. In short, always specify a default value.